README.md.tmpl

```diff
@@ -127,7 +127,7 @@ Here is the annotated config file, you only need to define this file if you wish
 
 # Supported resources
 {{range .supportedResources}}
-- {{. -}}
+- {{ .Type }}{{ if not .Tested }} *(untested)*{{ end -}}
```

Comment:
"untested" isn't very clear, but I was unsure on a better wording. Perhaps untested resources could be moved to a different section (e.g. "We support these resources, but because of {reasons}, we do not verify support in our integration tests.")